src/app/cdk/side-bar/modals/modal-websites/modal-websites.component.ts

```diff
@@ -99,7 +100,7 @@ export class ModalWebsitesComponent implements OnInit, OnDestroy {
       group[website.putCode] = new FormGroup({
         description: new FormControl(website.urlName),
         url: new FormControl(website.url.value, {
-          validators: [Validators.pattern(URL_REGEXP)],
+          validators: [Validators.required, Validators.pattern(URL_REGEXP)],
```

Comment:
Our `URL_REGEXP` does allow URLS like `google.com` the backend requires a complete email. 
We need to either add a backend async validator or add a new frontend local validation